### ISSUE TITLE
Digist Authentification

### ISSUE THREAD

**Issue by menschmachine-de** [ccrisan/motioneye#628]
Hi,

I would like to use motioneye with a Dahua IP Camera

VLC plays nice.
The Dahua has only Digist-Auth .
Does motioneye support Digist-Auth.

Could find anything
Best regards

**Comment by ccrisan** [comment]
Yes, it does, if you're talking about adding an IP camera that requires digest authentication. Make sure you're using motion-4.x.

**Comment by dec0dOS** [comment]
Getting same error with Dahua IP camera. 
<img width="428" alt="Снимок экрана 2019-09-13 в 14 51 42" src="https://user-images.githubusercontent.com/14824142/64860597-4c686980-d636-11e9-9350-1575e1c11acb.png">

**Comment by WarriorXK** [comment]
Works for me when using RTSP like so: rtsp://192.168.1.51:554/live

**Comment by dec0dOS** [comment]
RTSP is too CPU consuming so JPEG is more preferable.
